config/kubectl/crd/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../crd
+
+# Adds namespace to all resources.
+namePrefix: wlo-
+
+# Labels to add to all resources and selectors.
+commonLabels:
+  app.kubernetes.io/instance: websphere-liberty-operator
+  app.kubernetes.io/managed-by: olm
```

Comment:
`app.kubernetes.io/managed-by: olm` label should be removed here and elsewhere (below) since it's not applicable to the kubectl-install option